src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -1,6 +1,5 @@
 <form [formGroup]="form" (ngSubmit)="saveLayout($event)" novalidate class="dot-edit-layout__form">
```

Comment:
1. [this](https://user-images.githubusercontent.com/751424/34542227-59a0a38c-f0a1-11e7-8750-9c8fcc79de0e.png) and [this](https://user-images.githubusercontent.com/751424/34542272-86e0da4c-f0a1-11e7-9a79-db49d94f4c4c.png) still broken
2. action button should be 32px and have margin, check zeplin.